pages/_app.js

```diff
@@ -1,14 +1,16 @@
 import GlobalStyle from "@/styles";
 import Head from "next/head";
-
+import DiscoverPage from "./feed";
+import Navbar from "./navbar";
 export default function App({ Component, pageProps }) {
   return (
     <>
       <GlobalStyle />
       <Head>
         <title>Capstone Project</title>
       </Head>
-      <Component {...pageProps} />
+      <DiscoverPage />
```

Comment:
You should not insert a specific page here. The previous line was correct:
```
<Component {...pageProps} />
```
This will make to the Next.JS routing mechanism work. So users will see the "DiscoverPage" when they navigate to "/feed" in the browser